websites/A/AniMixPlay/presence.ts

```diff
@@ -1,145 +1,125 @@
 const presence = new Presence({
-    clientId: "787739407720513596"
-  }),
-  strings = presence.getStrings({
-    play: "presence.playback.playing",
-    pause: "presence.playback.paused",
-    browsing: "presence.activity.browsing"
-  });
+  clientId: "787739407720513596"
+}),
+strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused",
+  browsing: "presence.activity.browsing"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
 
-function getTimestamps(
-  videoTime: number,
-  videoDuration: number
-): Array<number> {
-  const startTime = Date.now(),
-    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
-  return [Math.floor(startTime / 1000), endTime];
-}
-const browsingTimer = Math.floor(Date.now() / 1000);
-
-let timestamps,
-  video,
-  iFrameVideo: boolean,
-  currentTime: number,
-  duration: number,
-  paused: boolean,
-  playback;
+let endTimestamp,
+video,
+iFrameVideo: boolean,
+currentTime: number,
+duration: number,
+paused: boolean,
+playback,
+title: string,
+episode: string;
```

Comment:
or wait a minute - because these things are not assigned second time so I think we can do this in different way
```suggestion
playback;
```